Makefile-Standalone

```diff
@@ -111,7 +111,7 @@ ifeq ($(NO_OPENSSL),1)
 OPENSSL                         = no
 else
 ifeq ($(HOSTOS),darwin)
-OSX_OPENSSL_DIR                 = /usr/local/opt/openssl
+OSX_OPENSSL_DIR                 = $(shell brew --prefix)/opt/openssl
```

Comment:
Brew may not be installed (as in my case), so I believe you'll need to be a bit more defensive in programming this as was the case demonstrated on line 115.